spec/metadata.spec.js

```diff
@@ -0,0 +1,53 @@
+"use strict";
+
+describe("METADATA", () => {
+    var artifact, space;
+
+    beforeAll(() => {
+        return jasmine.shelf.createSpace().then((testSpace) => {
+            space = testSpace;
+        });
+    });
+    beforeEach(() => {
+        return space.uploadArtifact("LOL").then((a) => {
```

Comment:
I know it probably doesn't matter, but I don't think you should use the variable name `a`. Maybe `testArtifact`? Similar to how you did it in the beforeAll.